docs/linux/reporting_kernel_bugs.md

```diff
@@ -1,11 +1,39 @@
 # Reporting Linux kernel bugs
 
-Before reporting a bug make sure nobody else already reported it. The easiest way to do this is to search through the [syzkaller mailing list](https://groups.google.com/forum/#!forum/syzkaller), [syzkaller-bugs mailing list](https://groups.google.com/forum/#!forum/syzkaller-bugs) and [syzbot dashboard](https://syzkaller.appspot.com/upstream) for key frames present in the kernel stack traces.
-
-Please report found bugs to the Linux kernel maintainers.
+Before reporting a bug **make sure nobody else has already reported it**. The easiest way to do
+this is to search through the [syzkaller mailing list](https://groups.google.com/forum/#!forum/syzkaller),
+[syzkaller-bugs mailing list](https://groups.google.com/forum/#!forum/syzkaller-bugs),
+[syzbot dashboard](https://syzkaller.appspot.com/upstream),
+and [kernel mailing lists](https://lore.kernel.org/) for key frames present in the kernel stack traces.
+
+**Do NOT report bugs on old kernel versions**.
+Old kernel generally means older than a week. Reports on old kernels are usually negatively accepted,
+and asked to be reproduced on a fresh kernel. Do not report bugs found on stable/LTS kernels.
+Bug on stable/LTS kernels should be reproduced on a fresh upstream kernel, and reported as such.
+Or, if they are already fixed upstream, the fixing patch should be
+[submitted to stable](https://www.kernel.org/doc/html/next/process/stable-kernel-rules.html).
+
+Please **report bugs to the Linux kernel maintainers**.
 To find out the list of maintainers responsible for a particular kernel subsystem, use the [get_maintainer.pl](https://github.com/torvalds/linux/blob/master/scripts/get_maintainer.pl) script: `./scripts/get_maintainer.pl -f guilty_file.c`. Please add `syzkaller@googlegroups.com` to the CC list.
-Make sure to mention the exact kernel branch and revision where the bug occurred.
-Many kernel mailing lists reject HTML formatted messages, so use the plain text mode when sending the report.
+
+**Minimal info** to include in the report:
+ - exact kernel branch and revision where the bug occurred
+ - exact kernel .config
+ - kernel OOPS message (BUG, KASAN report, etc), preferably with source files and line numbers
+ - reproducer, if known (see below)
+
+**Properly configure the kernel**.
+Improperly-configured kernel may lead to bad, and even false reports. There is no official way to get
+proper config for testing/fuzzing. You need to tune timeouts, enable debug info, enable lots of debug configs,
+disable some other debug configs, etc. You may use one of
+[syzbot configs](/dashboard/config/linux/upstream-apparmor-kasan.config), or check guidelines for
+[individual settings](/dashboard/config/linux/bits).
+
+**Use the plain text** mode when sending the report.
```

Comment:
**Your email client should be configured in plain text mode** when sending bug reports.

Reply:
Done